app/controllers/talents_controller.rb

```diff
@@ -0,0 +1,47 @@
+class TalentsController < ApplicationController
+	before_action :set_talent, except: [:index, :new, :create]
```

Comment:
Watch out for your indentation 💣 . Make sure your editor has a "two spaces" configuration